GCTC-NTGC/gc-digital-talent
# Design - Update toast styles

## 👋 Introduction

We recently updated our alert styles, this makes similar design updates to the toasts to match those new alerts.

## ✅ TO DO

- [x] Replace existing toasts

## 🛠️ Usage

The new styles are used in the same way, the difference is where you import the `toast` function from.

```tsx
import { toast } from "@common/components/Toast" // Note: This is now being imported from common

toast.info("text");
toast.success("text");
toast.warning("text");
toast.error("text");
```

## 📷 Screenshot

<img width="538" alt="Screenshot 2022-11-14 at 10 57 00 AM" src="https://user-images.githubusercontent.com/4127998/201706043-d788c143-2bf5-4941-8a67-07dbe6eb78bd.png">

## 🧪 Testing

1. Run storybook `npm run storybook --workspace=common`
2. Confirm the toast story functions as expected
3. Navigate through the app, triggering toasts to confirm they have been updated
